uint8 mask

int16 Left
int16 Right

int16 FrontDepth
int16 RearDepth

int16 FrontTurn
int16 RearTurn
